### PR TITLE
Draft: Adds RSS feed for podcast episodes.

### PR DESCRIPTION
**Pull request by elland**
In order to accomplish this, it was necessary to break down the podcasts.html into individual pages for each episode, which we in turn can snapshot before applying the template to create an RSS feed.

There's more than a few things to complete, like adding artwork, a proper contact email, and other metadata. Just point me in the right direction, and I'll gladly update this with the missing content.

I'm also not 100% sure if everything has been correctly applied, someone more familiar with the code base please take a closer look.

### CODE REVIEW

File: site/podcast/episodes/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+../index.html
```
Comment (elland) [inline site/podcast/episodes/index.html:1]: For now it's just a symbolic link to the main podcasts index.

File: docs/podcast.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        </itunes:subtitle>
+        <itunes:duration>00:05:55
+        </itunes:duration>
+        <enclosure url="https://www.buzzsprout.com/1817535/8849953-00-teaser.mp3" length="355" type="audio/mpeg" />
+        <link>https://haskell.foundation/podcast/episodes/2021-07-16-episode-0-teaser.html
+        </link>
+        <description>
+          <![CDATA[
+            <div class="max-w-screen-xl mx-auto">
+  <div class="bg-gray-100 px-6 sm:px-12 lg:px-16 py-12 lg:py-24">
```
Comment (elland) [inline docs/podcast.xml:44]: Ideally should add a step to strip styles from the feed.

**Comment by kosmikus** [comment]
Thanks for trying to do this, but there already is an RSS feed generated from buzzsprout, which is accessible via the player, and we would prefer having one RSS feed rather than two potentially different ones.

If the buzzsprout feed is too difficult to find via the embedded player, we could just include a direct link to it.

**Comment by elland** [comment]
@kosmikus it was indeed hard to find, especially because the expectation is that it's listed on the main html. Since there's already an RSS feed being generated, linking it inside the website would be preferable, for sure.

There's also the issue of it not being listed on the iTunes directory, arguably the canonical directory for podcasts. Is that something I could also help with?

**Pull request closed without merging**